README.md

```diff
@@ -297,6 +298,18 @@ If you need to have the automapper available in your operation, you can
 implement the `MapperAwareInterface`, and use the `MapperAwareTrait`. The
 default `MapTo` and `MapFrom` operations use these.
 
+#### Dealing with polymorphic properties
```

Comment:
It might be clearer if the section comes after the `mapTo` section.

Reply:
I'm not sure I understand what you mean. Which `mapTo` section? it is after `mapTo` in the table

| MapTo | Maps the property to another class. Allows for [nested mappings](#dealing-with-nested-mappings). Supports both single values and collections. |
| MapToMultiple | Maps the property to list of another classes (maps to first found match). This can be used for [polymorphic properties](#dealing-with-polymorphic-properties). Allows for [nested mappings](#dealing-with-nested-mappings). Supports both single values and collections. |